### DISCUSSION TITLE
Cannot add Bitcoin to Custom currency Implementor feature

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#16319] [Feature requests]
### Operating System

Windows 11 Beta Insider

### OpenRCT2 build

f65e12f

### Describe the issue

₿ shows up as question mark in in game font. Can it be added so I can use bitcoin as a custom currency in my parks?

### Steps to reproduce

1. In options try to change currency to custom currency
2. copy and paste UTF-8 code from website for bitcoin symbol [₿]
3. Shows up as ? in in game menu

**Comment by Gymnasiast** [comment]
Moved to discussions as this is a feature request and not a bug.

**Comment by YJSoft** [comment]
It's expected since sprite font did not include all charactors.(especially emojis)
Minecraft have "Force Unicode Font" setting to always use ttf fonts(rather than sprite one) to avoid this kind of problem.
